src/app/modules/main/module.nim

```diff
@@ -50,7 +51,7 @@ import ../../core/eventemitter
 
 export io_interface
 
-type 
```

Comment:
@osmaczko first of all this PR is very hard to follow due to lot of changes like this one. So please revert all that and apply changes only you did for this particular task.

Reply:
Reverting is probably not necessary, but sure, maybe we can do the clean up in another PR next time

Reply:
It's "trim trailing whitespace on file save" IDE feature. Reverted.

I'll create separate PR removing trailing whitespaces across all sources.